composer.json

```diff
@@ -40,7 +40,7 @@
         "ircmaxell/random-lib": "^1.1",
         "jdorn/sql-formatter": "^1.2",
         "monolog/monolog": "^1.12",
-        "paragonie/random_compat": "^1.4",
+        "paragonie/random_compat": "^2.0",
```

Comment:
If we go this way, this line can just be dropped … You added it in #5366 to version lock.